qemu/tests/in_place_upgrade.py

```diff
@@ -90,8 +94,25 @@ def run(test, params, env):
         upgrade_test.post_upgrade_restore(test)
         # post checking
         upgrade_test.post_upgrade_check(test, post_release)
+        post_rhel_ver = upgrade_test.run_guest_cmd(check_rhel_ver)
         vm.verify_kernel_crash()
     finally:
-        if upgrade_test.session:
-            upgrade_test.session.close()
-        vm.destroy()
+        vm.graceful_shutdown(timeout=300)
+        try:
+            rt_dir = data_dir.get_data_dir() + "/"
+            image_format = params.get("image_format")
+            old_name = params.get("image_name")
+            old_name = "%s.%s" % (old_name, image_format)
```

Comment:
```suggestion
            image_name = params.objects("images")[0]
            image_params = params.object_params(image_name)
            old_name = storage.get_image_filename(
                data_image_params, data_dir.get_data_dir())
```

Reply:
Thanks for reviewing,.
Honestly,your way  need to import libs such as ,from virttest import qemu_storage,
from virttest import utils_disk and so on so forth, honestly, it's just a simple image backup&rename, your steps seem more complicated. Correct me if I was wrong, thank you !

Reply:
Updated, please help to review it  thanks. @PaulYuuu @fbq815